api/api_storage.go

```diff
@@ -46,6 +46,10 @@ type StorageMiner interface {
 	// SectorGetSealDelay gets the time that a newly-created sector
 	// waits for more deals before it starts sealing
 	SectorGetSealDelay(context.Context) (time.Duration, error)
+	// SectorSetExpectedSealDuration sets the expected time for a sector to seal
+	SectorSetExpectedSealDuration(context.Context, time.Duration) error
```

Comment:
We should probably create a new `Sealing` namespace for some of those functions, `Sector` is a bit weird here

Reply:
Yeah, strong agree...you okay with that being a separate PR?

Reply:
Yep